modules/ROOT/pages/rtf-deployment-settings.adoc

```diff
@@ -0,0 +1,73 @@
+= Configuring Runtime Fabric Deployment Settings
+
+You can deploy Partner Manager to an Anypoint Runtime Fabric configured with Amazon Elastic Kubernetes Service (EKS).
+
+== Prerequisites for Deploying to Runtime Fabric
+
+Before you deploy Partner Manager to Runtime Fabric, you must:
+
+* Ensure that you have the required permissions for using Runtime Fabric.
+* Create and configure the Runtime Fabric instance into which to deploy Partner Manager services, as described in https://developer.mulesoft.com/tutorials-and-howtos/runtime-fabric/runtime-fabric-aws-elastic-kubernetes-service[Getting Started with Runtime Fabric on Amazon Elastic Kubernetes Service (EKS)].
+* Configure Persistence Gateway in Runtime Fabric, as described in xref::docs-runtime-fabric:persistence-gateway.adoc[Persistence Gateway].
```

Comment:
Please add the hyperlink to persistence gateway doc: https://docs.mulesoft.com/runtime-fabric/2.4/persistence-gateway